package.json

```diff
@@ -2,7 +2,10 @@
   "name": "@adobe/aio-lib-events",
   "version": "1.1.3",
   "description": "Adobe I/O Events",
-  "repository": "https://github.com/adobe/aio-lib-events",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/adobe/aio-lib-events.git"
```

Comment:
Is this change intentional?

Reply:
got generated automatically when did `npm init` locally.

Reply:
ok